diff --git a/rules/rules.d/50-udev-default.rules b/rules/rules.d/50-udev-default.rules
index 3744707..3bfcad1 100644
--- a/rules/rules.d/50-udev-default.rules
+++ b/rules/rules.d/50-udev-default.rules
@@ -33,6 +33,7 @@ KERNEL=="winradio*", GROUP="video"
 
 # graphics
 KERNEL=="agpgart", GROUP="video"
+KERNEL=="card[0-9]*", GROUP="video"
 KERNEL=="pmu", GROUP="video"
 KERNEL=="nvidia*|nvidiactl*", GROUP="video"
 SUBSYSTEM=="graphics", GROUP="video"
